umbc-cmsc447-fa2021-voting/blind-voting-app
# #3, #7: Add user profile to the Django admin user page, Send emails on user creation and deletion

**NEW SETTINGS VARIABLE:** `DEFAULT_HOST` needs to be added to your settings file to generate email links. For testing I have `DEFAULT_HOST = 'http://localhost:8000'`.

To test w/ Django, run `python manage.py test`

To test for yourselves
1. Add a new user and verify in the console that an email has been sent with a reset password.
2. Delete the same user and verify in the console that a deletion email has been sent.
